src/components/header/Header.tsx

```diff
@@ -26,35 +24,40 @@ const Header = () => {
 
     //change id of the film for film page

```

Comment:
i would suggest using find instead of findIndex, and change full title match from '===' to 'title.contains(searchValue)'